lib/Onyx.js

```diff
@@ -1563,6 +1563,16 @@ function init({
     shouldSyncMultipleInstances = Boolean(global.localStorage),
     debugSetState = false,
 } = {}) {
+    Storage.init();
+
+    if (shouldSyncMultipleInstances) {
```

Comment:
Was there any specific reason to move this up here? I think it makes sense but I'm also surprised that it wasn't here to begin with.